### ISSUE TITLE
Add ability to sort collections

### ISSUE THREAD

**Issue by YellowAfterlife** [leafo/itch.io#231]
It would be nice to be able to sort collection order for own use and\or display on the page.

Currently the collections are shown in the reverse chronological order and that can be not so optimal both if you want to add games to a particular collection by default and if you want a particular collection to be shown first on the profile page.

**Comment by fasterthanlime** [comment]
So just to be 100% clear, that's *not* a duplicate of https://github.com/leafo/itch.io/issues/69, you're talking about the order in which different collections appear on your profile page, right?

**Comment by YellowAfterlife** [comment]
Yes, I'm talking about the collection order on the profile page. I've tried to make that clear, since stumbled upon issue 69 when searching.

**Comment by rakkarage** [comment]
:+1:
